src/app/content/utils.spec.ts

```diff
@@ -116,7 +116,7 @@ describe('getContentPageReferences', () => {
       },
       {
         bookId: '13ac107a-f15f-49d2-97e8-60ab2e3b519c',
-        bookVersion: undefined,
+        bookVersion: '29.7',
```

Comment:
@TomWoodward is there a scenario where a book would exist in the config but we would want this to be undefined?

Reply:
yea i think we want this to be undefined. i think it'll get populated downstream. with this test here i'm not sure how we were getting the wrong version though